beetroot_homeworks/Homework_13/V_controller.py

```diff
@@ -0,0 +1,78 @@
+CHANNELS = ["BBC", "Discovery", "TV1000"]
+INDEX_TO_NAME = dict(enumerate(CHANNELS, start=1))
+
+
+class TVcontroller:
+    carrent_index = 1
+
+    def __init__(self, INDEX_TO_NAME) -> None:
```

Comment:
I understand why `INDEX_TO_NAME` is in uppercase at line 2, but why is it in uppercase here? That's a different variable.